src/core/annotation.js

```diff
@@ -3775,24 +3788,33 @@ class FreeTextAnnotation extends MarkupAnnotation {
       );
       this.data.defaultAppearanceData.fontColor = fontColor;
       this.data.defaultAppearanceData.fontSize = fontSize || 10;
-    } else if (this._isOffscreenCanvasSupported) {
-      const strokeAlpha = params.dict.get("CA");
-      const fakeUnicodeFont = new FakeUnicodeFont(xref, "sans-serif");
-      this.data.defaultAppearanceData.fontSize ||= 10;
+    } else {
       const { fontColor, fontSize } = this.data.defaultAppearanceData;
-      this.appearance = fakeUnicodeFont.createAppearance(
-        this._contents.str,
+      this.data.textContent = this._contents.str.split(/\r\n?|\n/);
```

Comment:
Given that `this._contents.str` could very well contain an *empty* string, based on [this method](https://github.com/mozilla/pdf.js/blob/7873ad98bb3cf1e692dd83ba59ca2d4db584c946/src/core/annotation.js#L794-L802), is this actually correct here?

Furthermore, in the `extractTextContent` method we'll explicitly ignore an empty string given [this code](https://github.com/mozilla/pdf.js/blob/7873ad98bb3cf1e692dd83ba59ca2d4db584c946/src/core/annotation.js#L1219).

Hence, would it not be more appropriate to do something like the following here?
```suggestion
      if (this._contents.str) {
        this.data.textContent = this._contents.str.split(/\r\n?|\n/);
      }
```